HelpSource/Reference/ServerPluginAPI.schelp

```diff
@@ -0,0 +1,189 @@
+title:: Server Plugin API
```

Comment:
One problem is development overhead. We will have to at least partially roll our own system, because I doubt there are any existing systems that compile to SCDoc. Is it really necessary for a medium-sized API that changes at a glacial rate?